Frontend/voitheia-App/src/assets/i18n/de.json

```diff
@@ -156,6 +156,32 @@
       "Service": "Service",
       "Contact": "Kontakt"
     }
+  },
+
+  "Idea": {
+    "Texts": {
+      "Idea": "Die Idee",
+      "Text1": "Im Rahmen des des #WirvsVirus Hackathons der Bundesregierung im März 2020 haben wir zueinander gefunden – unter ganz besonderen Umständen. Ohne uns einmal persönlich getroffen zu haben, fingen wir an, uns einer besondere Challenge zu widmen: Wie können wir unsere Mitmenschen unterstützen, mit neuen Herausforderungen umzugehen, die Corona uns stellt: Einkaufen, obwohl man einer Risikogruppe angehört? Kinder betreuen und von zuhause unterrichten, und währenddessen das Homeoffice schmeißen? Und dann auch noch mit dem Hund Gassi gehen?",
+      "Text2": "Gleichzeitig gibt es doch so viele, die gerne helfen wollen, und dabei einfach nicht wissen, wie.",
+      "Vision": "Wir möchten Menschen zusammenbringen, die sich unterstützen und sich gemeinsam diesen Herausforderungen stellen."
+    }
+  },
+
+  "AboutUs": {
+    "Texts": {
+      "AboutUs": "Wer wir sind",
+      "Text1": "Wir sind ein Team aus 9 Entwicklern, Bastlern, kreativen Köpfen und enthusiastischen Weltverbesserern.",
+      "Lukas": "...der im Hintergrund die Fäden zieht",
+      "Andi": "...der im Hinterzimmer aufräumt",
+      "Sebastian": "...der sicherstellt, dass wir immer auf Wolke 7 schweben",
+      "Benedikt": "...der das Ganze nutzbar macht",
+      "Kevin": "...der Allrounder",
+      "Christoph": "...der uns mobil macht",
+      "Helen": "...die uns kritisch über die Schulter schaut",
+      "Sophie": "...die das Ganze mit Leben füllt"
+    }
```

Comment:
Can we include this inside of an "AboutUs" wrapper element? So that the file is better structured.